pkg/server/server_test.go

```diff
@@ -1341,3 +1330,17 @@ func NewFakeClientGetter(client client.Client) ClientGetter {
 func (g *FakeClientGetter) Client(ctx context.Context) (client.Client, error) {
 	return g.client, nil
 }
+
+type FakeKubeGetter struct {
+	kube kube.Kube
+}
+
+func NewFakeKubeGetter(kube kube.Kube) KubeGetter {
+	return &FakeKubeGetter{
+		kube: kube,
+	}
+}
+
+func (g *FakeKubeGetter) Kube(ctx context.Context) (kube.Kube, error) {
+	return g.kube, nil
+}
```

Comment:
Oh yeah! Can we move all the fake stuff out of test package and into a fakes package so that its importable and usable from go module consumers and their test infra too?

Reply:
I couldn't easily do that because of circular dependencies. I'll attempt to change that in one of the next PRs 🤔